packages/network-controller/tests/provider-api-tests/shared-tests.ts

```diff
@@ -287,7 +287,7 @@ export function testsForProviderType(providerType: ProviderType) {
       describe('net_version', () => {
         const networkArgs = {
           providerType,
-          infuraNetwork: providerType === 'infura' ? 'goerli' : undefined,
+          infuraNetwork: providerType === 'infura' ? 'sepolia' : undefined,
```

Comment:
Maybe if we move the `TESTNET` constant to `tests/helpers.ts` then we can reuse it here?